packages/terra-header/docs/DEPENDENCIES.md

```diff
@@ -0,0 +1,17 @@
+# Dependency Information
```

Comment:
We can delete this file, we no longer generate these.

Reply:
deleted here https://github.com/cerner/terra-core/pull/2545/commits/4aad68b62fb8d2aa13121a399e6038c3adf24de8